.github/pull_request_template.md

```diff
@@ -7,14 +7,10 @@ CHECKLIST
 
 # Summary
 
```

Comment:
why the changes to the PR template?

Reply:
Sorry. I can push these up as a separate commit. I was getting annoyed by those comments lol they are placed weird.